packages/api/src/hooks/useOnfidoNotificationEvent.ts

```diff
@@ -0,0 +1,28 @@
+import { useCallback } from 'react';
+import useMutation from '../useMutation';
+
```

Comment:
Previously I used `useQuery` for this, but it should be a mutation since we only execute this query when we want to upload the documents from Onfido to BE